linux/shim/client/client.go

```diff
@@ -80,7 +80,7 @@ func WithStart(binary, address, daemonAddress, cgroup string, debug bool, exitHa
 		if err = sys.SetOOMScore(cmd.Process.Pid, sys.OOMScoreMaxKillable); err != nil {
 			return nil, nil, errors.Wrap(err, "failed to set OOM Score on shim")
 		}
-		c, clo, err := WithConnect(address)(ctx, config)
+		c, clo, err := WithConnect(address, func() {})(ctx, config)
```

Comment:
could we just pass nil instead? I feel it'd be more natural and expected

Reply:
nil would cause a panic.  no-ops work well here

Reply:
It's just a matter of checking it before setting it though.

I just think it'd be more idiomatic.